data.json

```diff
@@ -872,6 +872,15 @@
     "dateAdded": "2020-10-02T12:46:00.000Z",
     "tags": ["stickers", "hacktoberfest", "expired"]
   },
+  {
+    "name": "Trigger.dev",
+    "difficulty": "medium",
+    "description": "Submit PRs in trigger.dev or job-showcase repos and earn points to redeem for swag.",
+    "reference": "https://trigger.dev/hacktoberfest",
+    "image": "https://i.imgur.com/kelNqon.jpg",
+    "dateAdded": "2023-10-21T23:15:00.000Z",
+    "tags": ["clothing", "hacktoberfest"]
```

Comment:
Please list the other offerings using existing tags if possible.

Reply:
sure i will do it now thank you for letting me know